nussl/separation/composite/overlap_add.py

```diff
@@ -0,0 +1,89 @@
+from librosa.core import audio
+from .. import SeparationBase
+from ... import AudioSignal
+import numpy as np
+import warnings
+import matplotlib.pyplot as plt
+
+class OverlapAdd(SeparationBase):
+    def __init__(self, separation_object, window_length=15, find_permutation=False):
+        """[summary]
+
+        Parameters
+        ----------
+        separation_object : SeparationBase
+            Separation object that overlap and add is applied to.
+        window_length : int, optional
+            Window length of overlap/add window, by default 15 seconds.
+            The hop length will be half the window length.
+        find_permutation : bool, optional
+            Whether or not to find the permutation between chunks before combining.
+        """
+        super().__init__(separation_object.audio_signal)
+
+        self.hop_length = window_length / 2
```

Comment:
No way for user to set hop length?

Reply:
Only `hop_length = window_length / 2` will work, I think.

Reply:
Yeah for COLA, but what if the user doesn't want COLA? What if they want to just apply a separator to non-overlapping chunks? That should be possible, no?